modules/InfiniteStories/lib/useInfiniteStoriesLoading.ts

```diff
@@ -45,13 +45,8 @@ export function useInfiniteStoriesLoading(
 
     const { canLoadMore, data, isLoading, loadMore, resetData } =
         useInfiniteLoading<StoryWithImage>({
-            fetchingFn: async (nextPage: number) => {
-                const { stories } = await fetchStories(
-                    nextPage,
-                    pagination.pageSize,
-                    category,
-                    currentLocale,
-                );
+            fetchingFn: async (nextPage: number, pageSize) => {
```

Comment:
No need for explicitly stating type here anymore, right?

Reply:
Correct 👍